Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs

```diff
@@ -294,6 +300,13 @@ private void OnOperativeZombified(EntityUid uid, NukeOperativeComponent componen
         RemCompDeferred(uid, component);
     }
 
+    // Starlight Start: Check round end when nukie is cuffed/uncuffed
+    private void OnNukieCuffStateChanged(EntityUid uid, NukeOperativeComponent component, ref CuffedStateChangeEvent args)
+    {
+        CheckRoundShouldEnd();
+    }
```

Comment:
You can simplify this.
```suggestion
    private void OnNukieCuffStateChanged(EntityUid uid, NukeOperativeComponent component, ref CuffedStateChangeEvent args) => CheckRoundShouldEnd();
```